app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryUpdate.kt

```diff
@@ -52,6 +52,12 @@ class BloodPressureEntryUpdate : Update<BloodPressureEntryModel, BloodPressureEn
         noChange()
       }
 
+      is DayChanged -> dispatch(HideDateErrorMessage)
```

Comment:
Can't we collapse these cases into one?

Reply:
```kotlin
is DayChanged, is MonthChanged, is YearChanged -> dispatch(HideDateErrorMessage)
```

Reply:
This is still being refactored. I will refactor the `Update` class once all the functionality is moved.